### PR TITLE
feat: implementation of scala fibonacci sample

### PR DESCRIPTION
**Pull request by octonato**
Not working yet hence draft. 
I tried to run service locally and it's not responding. So, probably some issue when converting?

### CODE REVIEW

File: samples/scala-fibonacci-action/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+For further details see [Running a service locally](https://developer.lightbend.com/docs/akka-serverless/developing/running-service-locally.html) in the documentation.
+
+## Exercise the service
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://docs.lbcs.dev/js-services/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
+
+```shell
+curl -XPOST -H "Content-Type: application/json" localhost:9000/fibonacci.Fibonacci/NextNumber -d '{"value": 5 }'
```
Comment (johanandren) [inline samples/scala-fibonacci-action/README.md:48]: Should be `/com.example.fibonacci.Fibonnaci/NextNumber` right?

File: samples/scala-fibonacci-action/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Fibonacci Action Service
+
+This project is based on the Akka Serverless Maven archetype.
```
Comment (johanandren) [inline samples/scala-fibonacci-action/README.md:3]: This is all copied and not correct for the Scala version

**Review by johanandren — COMMENTED** [review]
No proto file?

**Comment by RayRoestenburg** [comment]
Continuing here: https://github.com/lightbend/akkaserverless-java-sdk/pull/517

**Pull request closed without merging**